src/components/ZSplashScreen/ZSplashScreen.js

```diff
@@ -1,14 +1,32 @@
 import React from 'react';
+import {NavLink} from 'react-router-dom';
 import './ZSplashScreen.css';
 
 const ZSplashScreen = () => {
   return (
-    <div className='flex-splashscreen'>
-      <h1>Welcome to Zertify</h1>
-      <h2>The Edera Certification Generator</h2>
-      <img src='https://i.imgur.com/K27lINZ.png' alt='TemplatesChoice' />
-      {/* <img src='https://i.imgur.com/lFxUsdV.jpg' alt='TemplatesChoice2' />
-  <img src='https://i.imgur.com/bPBtyoZ.jpg' alt='TemplatesChoice3' /> */}
+    <div>
+      <div>
+        <h1 className='flex-splashscreen'>Welcome to Zertify</h1>
+        <h2 className='flex-splashscreen'>Certification Creator</h2>
+        <div className='instruction_btn'>
+          <NavLink to='/students'>
+            <button>Select elegible Student</button>
+          </NavLink>
+          <NavLink to='/form'>
+            <button>Add new Student/Course</button>
+          </NavLink>
+          <NavLink to='/help'>
+            <button>Help</button>
+          </NavLink>
```

Comment:
I would not put the "help" button on the Spalsh Screen. 

The user has enough info to make an action with the two buttons you already added. 
If the user is still lost, he/she can always refer to the navbar and click on the help icon.

Reply:
i agree camilee

Reply:
Agreed 👍